Application/EdFi.Ods.AdminApi/Infrastructure/Security/SecurityExtensions.cs

```diff
@@ -100,12 +101,46 @@ IWebHostEnvironment webHostEnvironment
                     IssuerSigningKey = signingKey
                 };
                 opt.RequireHttpsMetadata = !isDockerEnvironment;
+            })
+            .AddJwtBearer("IdentityProvider", options =>
+            {
+                var oidcIssuer = configuration.Get<string>("Authentication:OIDC:Authority");
```

Comment:
@jleiva-gap 
Should we add the statement
`options.RequireHttpsMetadata = configuration.Get<bool>("Authentication:OIDC:RequireHttpsMetadata");` 
here?